### PR TITLE
(MODULES-8098) - Add logconfig option to config file

### PR DESCRIPTION
**Pull request by eimlav**

### CODE REVIEW

**Comment by tphoney** [comment]
Readme change too ?

**Comment by tphoney** [comment]
dumb question, are the docs in the reference coming from puppet strings ?

**Comment by eimlav** [comment]
Yep! That burst entry that was added must just be due to Puppet Strings not being ran since it was added.

**Comment by tphoney** [comment]
Brillo !!! @eimlav

File: manifests/config.pp [hunk 1]
```diff
@@ -90,7 +90,7 @@
 
   if $ntp::disable_dhclient {
     augeas { 'disable ntp-servers in dhclient.conf':
-      context => '/files/etc/dhcp/dhclient.conf',
+      context => '/etc/dhcp/dhclient.conf',
```
Comment (raphink) [inline manifests/config.pp]: @eimlav Why was this changed? Maybe you meant to use `incl` instead of `context`? As it is, it seems to be currently broken.

Comment (raphink) [reply manifests/config.pp]: https://github.com/puppetlabs/puppetlabs-ntp/pull/521

Comment (eimlav) [reply manifests/config.pp]: Hi @raphink. I'm not sure why I made this change... I'll put a fix up to revert it back!
